common/sgx/verifier.c

```diff
@@ -390,6 +394,22 @@ static oe_result_t _fill_with_known_claims(
 
     if (format_type != SGX_FORMAT_TYPE_LOCAL)
     {
+        // TCB status.
+        OE_CHECK(oe_sgx_add_claim(
+            &claims[claims_index++],
+            OE_CLAIM_TCB_STATUS,
+            sizeof(OE_CLAIM_TCB_STATUS),
+            tcb_status,
+            strlen(tcb_status) + 1));
+
+        // TCB date.
+        OE_CHECK(oe_sgx_add_claim(
+            &claims[claims_index++],
+            OE_CLAIM_TCB_DATE,
+            sizeof(OE_CLAIM_TCB_DATE),
```

Comment:
Since these are named as not SGX-specific, can you map them to some name in either https://datatracker.ietf.org/doc/draft-birkholz-rats-suit-claims/ or https://datatracker.ietf.org/doc/draft-ietf-rats-eat/ ?
Do you think there's a gap in those documents that there should be a github issue opened to address, or do you think that this PR needs to be updated to match some term in those docs?

Reply:
`TCB status` indicates the security patching level of firmware and software. I think the best match would be the Security Level Claim in https://datatracker.ietf.org/doc/draft-ietf-rats-eat/ 3.9. "This claim characterizes the device/entity ability to defend against attacks...". So how about naming two new claims as `OE_CLAIM_SECURITY_LEVEL` and `OE_CLAIM_SECURITY_LEVEL_ISSUE_DATE`?

Reply:
Per SIG-Attestation meeting discussion, TCB_STATUS and TCP_DATE are actually claims in an Endorsement, whereas EAT is currently scoped to Evidence and Attestation Results, so they need not map directly.   TCB Level, on the other hand, is what is in Evidence and can map to the Version claim in EAT (currently in https://datatracker.ietf.org/doc/draft-birkholz-rats-suit-claims/  section 3.1.8 until it gets merged into the main EAT spec).   Currently OE does not expose the TCB level in a claim, but it should in the future.  This can be done in a separate PR, but we should create a github issue to track it.

Reply:
Filed #4033